drivers/vulkan/vulkan_context.cpp

```diff
@@ -493,6 +493,8 @@ Error VulkanContext::_create_physical_device() {
 	//  features based on this query
 	vkGetPhysicalDeviceFeatures(gpu, &physical_device_features);
 
+	physical_device_features.robustBufferAccess = false; //turn on robust buffer access, which can hamper performance on some hardware
```

Comment:
That's turn *off*, not on as the comment says.

Reply:
how could I screw up a oneliner, I will go hide in shame.